dags/map_reproducibility/utils/common_utils.py

```diff
@@ -798,14 +797,24 @@ def get_nemo_metrics_cmds(
 
 
 def cleanup_all_runs_cmds(cluster, cluster_region, prefix="cml-"):
-  cleanup_cmds = (
-      f"echo 'Getting credentials for cluster {cluster}...' && gcloud container clusters get-credentials {cluster} --region {cluster_region} --project {PROJECT}",
+  get_credentials_cmd = (
```

Comment:
still some duplication in lines 800 - 805. I'm guessing its because of the "--project" flag ? is this something we could add to the get_credentials_command() function ? either always specify it or based on a parameter ?